backend/start.py

```diff
@@ -79,5 +79,23 @@ def count():
     return jsonify(len(data.get('entries')))
 
 
+@app.route('/api/categories/trending')
+@cross_origin
```

Comment:
This endpoint is not working. The reason is because you missed the brackets 
`@cross_origin` -> `@cross_origin()`